dev-libs/icu/icu67-67.1.recipe

```diff
@@ -129,8 +129,8 @@ BUILD()
 	cd source
 
 	# Created during build and prevent rebuilds.
-	#rm -rf tools/tzcode/vanguard
-	#cp ../../../sources-3/tzdata*.tar.gz tools/tzcode/
+	rm -rf tools/tzcode/vanguard
+	cp ../../../sources-3/tzdata*.tar.gz tools/tzcode/
```

Comment:
These were disabled, is there any reason to enable this now? (maybe they should be enabled for future versions if there is a benefit from it)  :)

Reply:
on older icu version the included tzdata is too old. so the recipe uses another source for tzdata and puts it in the right place for the build to take into account. This isn't needed on newest versions because they include up to date tzdata.